client/my-sites/customer-home/cards/features/webinar-promotion/index.jsx

```diff
@@ -0,0 +1,46 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { Card } from '@automattic/components';
+import { isDesktop } from '@automattic/viewport';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import CardHeading from 'components/card-heading';
+import ExternalLink from 'components/external-link';
+import temporaryIllustration from 'assets/images/customer-home/illustration--task-connect-social-accounts.svg';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+export const WebinarPromotion = () => {
+	const translate = useTranslate();
+
+	return (
+		<Card className="webinar-promotion">
```

Comment:
Oh, we now have a `Task` component that can serve as the base of these commonly-styled cards. See `/cards/tasks/connect-accounts` for an example usage 👍

Reply:
Forgive me if I'm missing something obvious here, but isn't that intended for the wider tasks that use the `action-panel`, like below, rather than the smaller card which this file renders?

<img width="1099" alt="Screenshot 2020-04-28 at 10 12 52" src="https://user-images.githubusercontent.com/43215253/80469709-d69cde00-8938-11ea-91be-8f69c373f0cb.png">